onnx/external_data_helper.py

```diff
@@ -185,10 +185,22 @@ def _get_all_tensors(onnx_model_proto: ModelProto) -> Iterable[TensorProto]:
                  _get_attribute_tensors(onnx_model_proto))
 
 
+def _get_initializer_tensors_from_graph(onnx_model_proto_graph: GraphProto) -> Iterable[TensorProto]:
+    """Create an iterator of initializer tensors from ONNX model graph."""
+    for initializer in onnx_model_proto_graph.initializer:
+        yield initializer
+    for node in onnx_model_proto_graph.node:
+        for attr in node.attribute:
+            if attr.type == AttributeProto.AttributeType.GRAPH:
+                yield from _get_initializer_tensors_from_graph(attr.g)
+            if attr.type == AttributeProto.AttributeType.GRAPHS:
```

Comment:
```suggestion
            if attr.type == AttributeProto.GRAPHS:
```
Mypy typecheck failed. Using AttributeProto.GRAPHS should help. L194 ditto.

Reply:
Thanks for reviewing it. I'll add this functionality to attribute tensors as well.